datashare-app/src/main/java/org/icij/datashare/web/RootResource.java

```diff
@@ -72,7 +74,9 @@ public String getRoot(Context context) throws IOException {
      */
     @Get("settings")
     public Map<String, Object> getPublicSettings() {
-        return propertiesProvider.getFilteredProperties(".*Address.*", ".*Secret.*", ".*Url.*", ".*Key.*");
+        Map<String, Object> filteredProperties = propertiesProvider.getFilteredProperties(".*Address.*", ".*Secret.*", ".*Url.*", ".*Key.*");
+        filteredProperties.put("pathSeparator", File.separator);
```

Comment:
Just to be sure, should it belong to the properties or it ok if it's just computed ?

Reply:
Better if it's computed!